src/test/config/bitbucket/helm_parameters

```diff
@@ -4,7 +4,7 @@ CHART_TEST_VALUES_BASEDIR=${chart.testValues.basedir}
 PRODUCT_NAME=bitbucket
 DB_NAME=bitbucket
 POSTGRES_CHART_VERSION=11.6.2
-POSTGRES_APP_VERSION=14
+POSTGRES_APP_VERSION=16
```

Comment:
I would have thought just this bump would have been sufficient, but my tests were still using PG 14 until I also bumped the version in deploy_app.sh